src/tryWith.js

```diff
@@ -0,0 +1,30 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * `tryWith` evaluates the first function; if it does not throw, it simply returns the
+ * result. If the first function *does* throw, `tryWith` evaluates the second function
+ * and returns its result. Note that for effective composition with this function, both
+ * the first and second functions should take the same types of arguments and return the
+ * same type of results.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig (x -> a) -> (x -> a) -> x -> a
+ * @param {Function} tryer The function that may throw.
+ * @param {Function} catcher The function that will be evaluated if `tryer` throws.
+ * @param {*} x
+ * @return {*} The result of evaluating `tryer` or possibly `catcher`.
+ * @example
+ *
+ *      R.tryWith(R.head, R.T, [false, false, false]); //=> false
+ *      R.tryWith(R.head, R.T, []); //=> true
```

Comment:
> Something like this?

That's what I would expect, yes, but mostly because that's how all similar functions in the library behave (including the new ``juxt`` function). Regarding composition, I can see how this function could be used as the first step in a pipeline, and then a non-unary case is as valid as it is for ``ifElse`` and friends.

> actually this doesn't throw, just returns undefined. need a better example.

``R.prop("foo", null)`` would do; only problem is that we already have ``propOr``.